application/tests/unit/common/models/UserTest.php

```diff
@@ -157,7 +157,9 @@ public function testUpdateProfileIfNeeded()
         $changed = $user->updateProfileIfNeeded($personnelData);
         $this->assertTrue($changed);
 
-        $personnelData->doNotDisclose = ! $personnelData->doNotDisclose;
+        $personnelData->hide === 'no'
```

Comment:
I'm not used to seeing assignments done in the right-hand sections of a ternary operator, so I'm wondering about readability here. What would you think of something like this instead?
```php
$personnelData->hide = ($personnelData->hide === 'no') ? 'yes' : 'no';
```
I'm tempted to recommend a new `getOppositeOf()` method, but that may be overkill. 🙂

Reply:
Yeah, that might be a little overkill. I did take your suggested rewrite, though.